ui/Screen/Settings.svelte

```diff
@@ -25,29 +43,80 @@
     min-width: var(--content-min-width);
     padding: 0 var(--content-padding);
   }
+
   section header {
     margin: 16px 0 24px 0;
     border-bottom: 1px solid var(--color-foreground-level-3);
-    padding: 12px;
+    padding: 12px 0;
```

Comment:
This breaks the alignment as Brandon illustrated here: https://github.com/radicle-dev/radicle-upstream/pull/629#issuecomment-655998993